molgenis-data/src/main/java/org/molgenis/data/util/MetaUtils.java

```diff
@@ -42,10 +41,7 @@ public static boolean isSystemPackage(@Nullable Package aPackage)
 		{
 			return false;
 		}
-		//Needs to run as system since you might not have permission to read the sys package
-		//Once we have READ_META permission checks on the packages this runAsSystem can be removed
-		//https://github.com/molgenis/molgenis/issues/7528
-		return runAsSystem(() -> aPackage.getId().equals(PACKAGE_SYSTEM) || (aPackage.getRootPackage() != null
-				&& aPackage.getRootPackage().getId().equals(PACKAGE_SYSTEM)));
+		return aPackage.getId().equals(PACKAGE_SYSTEM) || (aPackage.getRootPackage() != null
```

Comment:
Shouldn't this branch be tested in a JUnit test? There is also one for the EntityUtils class, but if you (as in the platform-team) have decided otherwise I could imagine this is not necessary.

Reply:
I’m juist reverting the RunAsSystem.